CHANGELOG-developer.asciidoc

```diff
@@ -27,6 +27,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2..v7.0.0-beta1[Check the H
 - Add (*common.Config).Has and (*common.Config).Remove. {pull}10363[10363]
 - Introduce ILM and IndexManagement support to beat.Settings. {pull}10347[10347]
 - Generating index pattern on demand instead of shipping them in the packages. {pull}10478[10478]
+- Add helper to get configurations from hints {pull}10952[10952]
```

Comment:
could you please remove this changelog? this change is not affecting usage or behavior